querydsl-core/src/test/java/com/querydsl/core/testutil/Parallelized.java

```diff
@@ -0,0 +1,40 @@
+package com.querydsl.core.testutil;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.runners.Parameterized;
+import org.junit.runners.model.RunnerScheduler;
+
+public class Parallelized extends Parameterized {
+
+    public Parallelized(Class<?> klass) throws Throwable {
+        super(klass);
+        setScheduler(new ThreadPoolScheduler());
+    }
+
+    private static class ThreadPoolScheduler implements RunnerScheduler {
+        private ExecutorService executor;
+
+        public ThreadPoolScheduler() {
+            int numThreads = Runtime.getRuntime().availableProcessors();
```

Comment:
The time is not that important to me, but how flaky it is / how much it fails.
If it works well with 4 then it should work well with 8.